### PR TITLE
Add Block footer in the ninth section

### PR DESCRIPTION
**Pull request by Flasto4ka**
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -477,6 +477,100 @@ <h2 class="course__title">Записаться на курс</h2>
...
+                  <img src="img/facebook 1.svg" alt="facebook" />
+                  <img src="img/instagram 1.svg" alt="instagram" />
+                  <img src="img/vk 1.svg" alt="vk" />
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="row">
+            <div class="footer__tems">
```
Comment (flast1k) [inline index.html:514]: it should be nav tag

File: index.html [hunk 2]
```diff
@@ -477,6 +477,100 @@ <h2 class="course__title">Записаться на курс</h2>
           </div>
         </div>
       </section>
+
+      <!-- Block footer -->
+
+      <section class="footer">
```
Comment (flast1k) [inline index.html:483]: footer instead of section

File: css/style.css [hunk 3]
```diff
@@ -630,3 +630,75 @@ ul {
...
+  display: flex;
+  gap: 80px;
+  margin-bottom: 60px;
+}
+
+.footer__line {
+  width: 32px;
+  height: 0px;
+  left: 511px;
+  top: 4398px;
```
Comment (flast1k) [inline css/style.css:667]: why do you need left and top here?

File: index.html [hunk 4]
```diff
@@ -477,6 +477,100 @@ <h2 class="course__title">Записаться на курс</h2>
...
+              <div class="column-lg-3">
+                <div class="footer__text-line">Политика конфиденциальности</div>
+              </div>
+
+              <div class="column-lg-3">
+                <div class="footer__text">support@hibrain.pro</div>
+              </div>
+
+              <div class="column-lg-3">
+                <div class="footer__text">8 800 000 00 00</div>
```
Comment (flast1k) [inline index.html:568]: `<a href="tel:8 800 000 00 00">8 800 000 00 00</a>`

File: index.html [hunk 5]
```diff
@@ -477,6 +477,100 @@ <h2 class="course__title">Записаться на курс</h2>
...
+              <div class="column-lg-3">
+                <div class="footer__text">© ООО «Лад Ай Ти», 2020</div>
+              </div>
+
+              <div class="column-lg-3">
+                <div class="footer__text-line">Политика конфиденциальности</div>
+              </div>
+
+              <div class="column-lg-3">
+                <div class="footer__text">support@hibrain.pro</div>
```
Comment (flast1k) [inline index.html:564]: `<a href="mailto: support@hibrain.pro">support@hibrain.pro</a>`

File: index.html (hunk 1 above)
Comment (Flasto4ka) [reply index.html:514]: +

File: css/style.css (hunk 3 above)
Comment (Flasto4ka) [reply css/style.css:667]: +

File: index.html (hunk 4 above)
Comment (Flasto4ka) [reply index.html:568]: +

File: index.html (hunk 5 above)
Comment (Flasto4ka) [reply index.html:564]: +

**Review by flast1k — APPROVED** [review]

**Pull request merged**